README.md

```diff
@@ -256,6 +256,58 @@ To check whether dnsmasq name server at 127.0.0.1 can resolve subdomains of dev.
 app.dev.gov.uk.		0	IN	A	127.0.0.1
 ```
 
+### How to: replicate data locally
+
+#### MySQL
+
+1. Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/mysql/?region=eu-west-1&tab=overview)
+
+2. Import the file into the local MySQL database, e.g. for Whitehall:
+
+```
+pv whitehall_production.dump.gz | gunzip | docker exec -i `docker ps | grep mysql | awk '{print $1}'` mysql -u root --password=root whitehall_development
+```
+
+#### PostgreSQL
+
+1. Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/postgres/?region=eu-west-1&tab=overview)
+
+2. Import the file into the local Postgres database, e.g. for Publishing API:
+
+
+```
+pv publishing_api_production.dump.gz  | gunzip | docker exec -i `docker ps | grep postgres | awk '{print $1}'` /usr/bin/psql -U postgres -qAt -d publishing-api
```

Comment:
I think the database should be dropped and then created, otherwise not all of the data is imported. Something like this should work

```bash
docker exec -i `docker ps | grep postgres | awk '{print $1}'` /usr/bin/psql -U postgres -qAt DROP DATABASE IF EXISTS "publishing-api"
docker exec -i `docker ps | grep postgres | awk '{print $1}'` /usr/bin/createdb -U postgres publishing-api
```